### PR TITLE
EBI survey banner in home page

### PR DESCRIPTION
**Pull request by gustavo-salazar**
Adding the banner details as a Callout on the Home page.

**NOTE:** Only to be merged the 20th May

### CODE REVIEW

File: src/pages/Home/index.js [hunk 1]
```diff
@@ -252,6 +254,23 @@ class Home extends PureComponent {
           />
         ) : null}
         <GeneralWarning />
+        <Callout type="warning">
+          <div>
+            Do data resources managed by EMBL-EBI and our collaborators make a
+            difference to your work?
+            <br /> Please take 10 minutes to fill in our{' '}
+            <Link
+              href="https://www.surveymonkey.com/r/HJKYKTT?channel=[webpage]"
```
Comment (matthiasblum) [inline src/pages/Home/index.js:263]: I suspect the `channel` param will be used to track the resources users were visiting when opening the survey.

```suggestion
              href="https://www.surveymonkey.com/r/HJKYKTT?channel=InterPro"
```

**Review by matthiasblum — APPROVED** [review]
LGTM.

If it's possible, it would be nice to have the URL of the survey in the config file, and only display the callout if the URL is not undefined or an empty string. This way, we can enable/disable the callout without having to change the code (just update the config file, build, and deploy).

File: src/pages/Home/index.js (hunk 1 above)
Comment (gustavo-salazar) [reply src/pages/Home/index.js:263]: I initially thought the same, but in the document the link changes in other parts as `channel[email]`, `channel[twitter]`, etc. so I leave it as `[website]`

Comment (matthiasblum) [reply src/pages/Home/index.js]: I noticed that too, but I thought the links be updated, e.g.:

- link in an email: `channel=recipient@company.com`
- link in a X/Twitter post: `channel=@interprodb`
- link in a LinkedIn post: `channel=interpro-pfam`

I'll reach the people in charge of the survey and ask them to clarify.

Comment (gustavo-salazar) [reply src/pages/Home/index.js]: cool, let's wait for that...  but in the meantime, I implemented your suggestion about the link in the config. Please have a look to it

**Review by matthiasblum — APPROVED** [review]
LGTM

File: src/pages/Home/index.js (hunk 1 above)
Comment (matthiasblum) [reply src/pages/Home/index.js]: Got confirmation that indeed we should leave it as it is.

**Pull request merged**